docs/topics/api/addons.rst

```diff
@@ -175,6 +175,7 @@ This endpoint allows you to fetch a specific add-on by id, slug or guid.
     :>json boolean is_disabled: Whether the add-on is disabled or not.
     :>json boolean is_experimental: Whether the add-on has been marked by the developer as experimental or not.
     :>json boolean is_featured: The add-on appears in a featured collection.
+    :>json boolean is_recommended: The add-on is recommended by Mozilla.
```

Comment:
Add it to changelog

Reply:
https://github.com/mozilla/addons-server/pull/11275/commits/0d23c4147daf414a2057bb574d3a487413dae91b